sdks/go/pkg/beam/core/runtime/graphx/translate.go

```diff
@@ -1376,3 +1398,13 @@ func UpdateDefaultEnvWorkerType(typeUrn string, pyld []byte, p *pipepb.Pipeline)
 	}
 	return errors.Errorf("unable to find dependency with %q role in environment with ID %q,", URNArtifactGoWorkerRole, defaultEnvId)
 }
+
+// UserStateCoderId returns the coder id of a user state
+func UserStateCoderId(ps state.PipelineState) string {
+	return fmt.Sprintf("val_%v", ps.StateKey())
+}
+
+// UserStateCoderId returns the key coder id of a user state
```

Comment:
```suggestion
// UserStateKeyCoderId returns the key coder id of a user state
```

Reply:
Thanks! Hopefully that saves a follow up linting PR 😅